controller/src/beerocks/master/db/db.cpp

```diff
@@ -2423,73 +2423,257 @@ const std::list<sChannelScanResults> &db::get_channel_scan_results(const sMacAdd
 //
 std::chrono::steady_clock::time_point db::get_client_persistent_data_last_edit(const sMacAddr &mac)
 {
-    return std::chrono::steady_clock::time_point::min();
+    auto n = get_node(mac);
+    if (!n) {
+        LOG(ERROR) << "node not found.... ";
+        return std::chrono::steady_clock::time_point::min();
+    } else if (n->get_type() != beerocks::TYPE_CLIENT) {
+        LOG(ERROR) << __FUNCTION__ << "node " << mac << " is not a valid client!";
+        return std::chrono::steady_clock::time_point::min();
+    }
+
+    return n->client_persistent_data_last_edit;
 }
 
 bool db::set_client_time_life_delay(const sMacAddr &mac,
                                     const std::chrono::seconds &time_life_delay_sec,
                                     bool set_persistent_db)
 {
+    auto n = get_node(mac);
+    if (!n) {
+        LOG(ERROR) << "node not found.... ";
+        return false;
+    } else if (n->get_type() != beerocks::TYPE_CLIENT) {
+        LOG(ERROR) << __FUNCTION__ << "node " << tlvf::mac_to_string(mac)
+                   << " is not a valid client!";
+        return false;
+    }
+
+    LOG(DEBUG) << __FUNCTION__ << ", time_life_delay_sec = " << time_life_delay_sec.count();
```

Comment:
If time_life_delay_sec is in a unit of seconds, Why do you use `count()` on it? It doesn't make sense.
I would expect that it will be really with a second unit.

Reply:
as we dicsussed offline, leaving it as is since the main usage is in the std::chrono:seconds() type